src/ts/components/transaction/Fee.tsx

```diff
@@ -2,10 +2,24 @@ import React from 'react'
 import t from '../../services/i18n'
 import { connect } from 'react-redux'
 import { IAppState } from '../../background/store/all'
-import { ChainProperties } from '@polkadot/types'
+import { ChainProperties, Balance } from '@polkadot/types'
 import ApiPromise from '@polkadot/api/promise'
-import { formatBalance } from '@polkadot/util'
+import { compactToU8a, formatBalance } from '@polkadot/util'
 import styled from 'styled-components'
+import { IExtrinsic } from '@polkadot/types/types'
+import { DerivedFees } from '@polkadot/api-derive/types'
+import BN = require('bn.js')
+
+const LENGTH_PUBLICKEY = 32 + 1 // publicKey + prefix
+const LENGTH_SIGNATURE = 64
+const LENGTH_ERA = 1
+export const SIGNATURE_SIZE = LENGTH_PUBLICKEY + LENGTH_SIGNATURE + LENGTH_ERA
+
+export const calcSignatureLength = (extrinsic?: IExtrinsic | null, accountNonce?: BN): number => {
```

Comment:
we probably want to call this `calcExtrinsicLength` as it is used to get the extrinsic length.

Reply:
I am not sure though. I borrowed this function from polkadot-js/apps, and it adds signature size as a constant.